src/main/java/com/GeorgesServer/app/Router.java

```diff
@@ -22,6 +23,9 @@ public IHandler route(String publicFolderPath, ClientRequest request) {
         if (request.getMethod().equals("POST")) {
             return new PostHandler(publicFolderPath, request);
         }
+        if (request.getMethod().equals("OPTIONS")) {
```

Comment:
Might want to start thinking about a more polymorphic solution here to get to prevent a proliferation of conditionals